src/gestures.c

```diff
@@ -312,10 +312,11 @@ static void buttons_update(struct Gestures* gs,
 				foreach_bit(i, ms->touch_used) {
 					timeraddms(&ms->touch[i].down, cfg->button_expire, &expire);
 					if ((cfg->button_move || cfg->button_expire == 0 || timercmp(&ms->touch[latest].down, &expire, <)) &&
-                            !(GETBIT(ms->touch[i].state, MT_THUMB) && cfg->ignore_thumb) &&
-                            !(GETBIT(ms->touch[i].state, MT_PALM) && cfg->ignore_palm)) {
+						!(GETBIT(ms->touch[i].state, MT_THUMB) && cfg->ignore_thumb) &&
+						!(GETBIT(ms->touch[i].state, MT_PALM) && cfg->ignore_palm) &&
+						!(GETBIT(ms->touch[i].state, MT_EDGE))) {
 						touching++;
-                    }
+					}
```

Comment:
These lines were all indented with spaces, so I've changed them to tabs to match the style in the rest of this file.
![screenshot from 2016-05-26 17-34-03](https://cloud.githubusercontent.com/assets/2053039/15590904/11d46aaa-2368-11e6-8cf5-6488107f69fd.png)